frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -60,6 +64,42 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
         tierDisplayOptions.push({ label: `Current bill`, value: 'total' })
     }
 
+    const showPipelineAddonNotice =
+        addon.type === 'data_pipelines' &&
+        addon.subscribed &&
+        featureFlags['data-pipelines-notice'] &&
+        addon.plans?.[0].plan_key === 'addon-20240103-og-customers'
+
+    if (showPipelineAddonNotice) {
+        setProductSpecificAlert({
+            status: 'info',
+            title: 'Welcome to the data pipelines addon!',
+            message: `We've moved data export features (and cost) to the data pipelines addon to better reflect user needs. Your overall
+                    price hasn't changed.`,
+            action: {
+                onClick: () => {
+                    posthog.capture('data pipelines notice clicked')
+                    // if they don't dismiss it now, we won't show it next time they come back
+                    posthog.capture('data pipelines notice dismissed', {
```

Comment:
Will clicking this actually dismiss the banner?

Reply:
Just saw the note about the feature flag

Reply:
Yeah, the dismissal is actually handled automatically with the lemonBanner dismiss key. The event and property is just for the flag, which prevents it from blocking other billing banners in the future.